### PR TITLE
Update storybook

### PR DESCRIPTION
**Pull request by wps13**
**What**: Update the storybook dependencies, add knobs and update other dependencies.

**Why**: Some dependencies were outdated and some where missing, so fix this problem. Also add knobs for live edition of the components in the storybook page.

**How**: Installed the add-on knobs, inserted it into the stories, adding variables for default values and using it as default for color and boolean functions.

### CODE REVIEW

File: stories/ball.stories.tsx [hunk 1]
```diff
@@ -35,27 +35,217 @@ const style = {
...
+        return (
+            <div style={style}>
+                <BallRotate
+                    color={color('Color', defaultColor)}
+                    loading={boolean('loading', true)}
+                />
+            </div>
+        );
+    })
+    .add('Scale', () => {
```
[the remark this answers is not in the document]
Comment (wps13) [reply stories/ball.stories.tsx]: The local lint doesn't show this error.

**Comment by wps13** [comment]
@jameswlane  you can leave the codacy reviews, i will check them later

**Comment by jameswlane** [comment]
@wps13 Do you think we should get an actual domain name for this project I can grab one today and set it up on Zeit? If so any suggestions?

**Comment by jameswlane** [comment]
@wps13 Back at my laptop 🎉 fixed the last few linting errors 👍 I think we are getting there.

There is an npm script `npm run verify` that you can run before committing that should check for any errors.

**Comment by wps13** [comment]
@jameswlane I am currently refactoring the stories, creating them dynamically.

**Comment by jameswlane** [comment]
@wps13 Whoops, I apologize. I was just trying to get the last check to pass while I had a minute. I will leave the stories alone, thank you. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by wps13** [comment]
@jameswlane No problem! I'm glad to see that you are back :) Thanks for the help, anyway.

**Comment by jameswlane** [comment]
@wps13 Checking the deployed site, it seems to be night finding React.

https://react-pure-loaders-gv3w8i9k8.now.sh/?path=/story/ball-loaders--beat

**Comment by wps13** [comment]
@jameswlane Just fixed that.

**Pull request merged**